content/en/docs/troubleshooting.md

```diff
@@ -11,92 +11,89 @@ To suggest a troubleshooting item, [create a pull request PR with your suggested
+            ├── digest: sha256:cae906b84806687f02272d00a7000ff31a6be6281bb72f16bdef2fcd67d41786
+            └── size: 942
 ```
 
 The output contains all the certificate information used for signing the artifact. You can use this information to validate the signing certificates and certificate chain.
 
-The output also contains details about the artifact that was signed, such as the digest. You can use this information to confirm the correct artifact was signed.
+The output also contains details about the artifact that was signed, such as the digest. You can use this information to confirm that the correct artifact was signed.
 
 
 ## Enabling notation CLI commands logging
 
-Enabling a more detailed logging using `--verbose` can help troubleshoot issues with the notation CLI. Using `--verbose` or `-v` gives more detailed output for the notation CLI commands that can be used for troubleshooting.
+To troubleshoot issues with the notation CLI, you can use the `--verbose` or `-v` flag for more detailed logging. Using the `--verbose` flag gives more detailed information about the notation CLI command. This is useful in identifying and resolving issues effectively. 
 
-When running a notation CLI command, use the `--verbose` flag to enable verbose logging. For example:
+Use the `--verbose` flag to enable verbose logging when running a notation CLI command. For example:
 
-```shell
-notation verify --verbose localhost:5000/net-monitor@sha256:sha256:xxx
+```console
+notation verify --verbose localhost:5000/net-monitor@sha256:xxx
 ```
 
 In addition, there is `--debug`, which is intended for developers to debug the notation CLI. It is not recommended to use `--debug` for end-user troubleshooting.
 
 ## When I verify an artifact, I get the error 'Error: open $HOME/.config/notation/trustpolicy.json: no such file or directory'
 
-This error is likely related to trust policy configuration. Verify you have a trust policy set up before you attempt to verify an artifact. For more details, see [Manage trust policies]({{< ref "/docs/how-to/manage-trust-policy" >}}
+This error is likely related to trust policy configuration. Verify you have a trust policy set up before you attempt to verify an artifact. For more details, see [Manage trust policies]({{< ref "/docs/how-to/manage-trust-policy" >}})
 
 ## When I verify an artifact, I get the error '"$HOME/.config/notation/truststore/x509/ca/mytruststore" does not exist'
 
 This error indicates the trust store doesn't exist or the trust store name is not correct. Trust store typically contains a set of certificate files, where the trust identities are retrieved to verify signatures. You can use `notation cert add` to add trust stores.
 
-The above error shows the type of trust store is `ca`, and the trust store name is `mytruststore`. 
+The above error shows that the type of trust store is `ca`, and the trust store name is `mytruststore`. 
```

Comment:
I think it will be good to callout that there are other types if trust stores as well, signingAuthority and tsa. We can even link to the spec - https://github.com/notaryproject/notaryproject/blob/5afbf9983756133dec85b7e8a323349fa10e3e56/specs/trust-store-trust-policy.md?plain=1#L52

Reply:
Thanks for pointing that out. I will make sure to include it.